### ISSUE TITLE
Error: Attempt to postMessage on disconnected port

### ISSUE THREAD

**Issue by angela-d** [kkapsner/CanvasBlocker#382]
## Description

Usability blocking when this error surfaces in console; occurs on both PRTG web dashboard and ally.com

Canvas Blocker has to be disabled for these sites in order to use them.

## Expected Behavior

On ally.com, for example, clicking Log In should bring a slider with login fields.

## Current Behavior

With Canvas Blocker enabled, a black transparency layer appears and nothing else.

## Possible Solution

Bugfix

## Steps to Reproduce (for bugs)

1. Go to ally.com
2. Click log in
3. No login slider appears
4. Similar behavior on PRTG web dashboard while viewing sensor logs and statuses.. possibly caused by similar JS libraries?

## Context

Just by visiting the sites with an everyday profile.

## Your Environment

* CanvasBlocker Version used: 0.5.11
* Firefox version incl. 32- or 64-bit: Waterfox 56.2.12 64-bit
* Operating System and version (desktop or mobile): Debian 9.9
* Installed addons:
CanvasBlocker
Classic Add-ons Archive
Cookie AutoDelete
Decentraleyes
GNOME Shell integration
Invidious Redirect
Multi-Account Containers
NoScript
Old Reddit Redirect
Privacy Badger
Status-4-Evar
Text Contrast for Dark Themes
uBlock Origin
uMatrix

## Your Settings

```text
{
	"logLevel": 1,
	"urlSettings": [
		{
			"url": "^https://www\\.google\\.com/recaptcha/api2/(?:b?frame|anchor).*$",
			"protectWindow": false
		}
	],
	"hiddenSettings": {},
	"expandStatus": {},
	"displayHiddenSettings": false,
	"whiteList": "",
	"sessionWhiteList": "",
	"blackList": "",
	"blockMode": "fake",
	"protectedCanvasPart": "readout",
	"minFakeSize": 1,
	"maxFakeSize": 0,
	"rng": "nonPersistent",
	"protectedAPIFeatures": {},
	"useCanvasCache": true,
	"ignoreFrequentColors": 0,
	"minColors": 0,
	"fakeAlphaChannel": true,
	"persistentRndStorage": "",
	"persistentIncognitoRndStorage": "",
	"storePersistentRnd": false,
	"persistentRndClearIntervalValue": 0,
	"persistentRndClearIntervalUnit": "days",
	"lastPersistentRndClearing": 1562980960827,
	"sharePersistentRndBetweenDomains": false,
	"askOnlyOnce": "individual",
	"askDenyMode": "block",
	"showCanvasWhileAsking": true,
	"showNotifications": true,
	"highlightPageAction": "none",
	"highlightBrowserAction": "color",
	"displayBadge": true,
	"storeNotificationData": false,
	"storeImageForInspection": false,
	"ignoreList": "",
	"ignoredAPIs": {},
	"showCallingFile": false,
	"showCompleteCallingStack": false,
	"enableStackList": false,
	"stackList": "",
	"protectAudio": true,
	"audioFakeRate": "100",
	"audioNoiseLevel": "minimal",
	"useAudioCache": true,
	"audioUseFixedIndices": true,
	"audioFixedIndices": "24",
	"historyLengthThreshold": 2,
	"protectWindow": true,
	"protectDOMRect": true,
	"domRectIntegerFactor": 4,
	"blockDataURLs": true,
	"protectNavigator": false,
	"navigatorDetails": {},
	"displayAdvancedSettings": true,
	"displayDescriptions": false,
	"theme": "auto",
	"dontShowOptionsOnUpdate": true,
	"isStillDefault": false,
	"storageVersion": 0.5
}
```

**Comment by kkapsner** [comment]
This problem should be solved with 0.5.12 - I just forgot to mark it as compatible with Firefox 56 (which Waterfox is based on). Please check if you now get the update.

**Comment by angela-d** [comment]
Yeah!

Updated today and I now see the slider, thanks!
